api/controllers/dapps.js

```diff
@@ -36,6 +36,19 @@ DappsController.getDapps = function (context, next) {
 		try {
 			if (err) { return next(err); }
 
+			data = _.map(data, function (dapp) {
+				if (_.isNull(dapp.description)) {
+					dapp.description = '';
+				}
+				if (_.isNull(dapp.tags)) {
+					dapp.tags = '';
+				}
+				if (_.isNull(dapp.icon)) {
+					dapp.icon = '';
+				}
+				return dapp;
+			});
+
```

Comment:
@diego-G Its preferred to clone the data response first before making any change to it. We can came up with scenario that some variable is passed as reference form underlying modules and changing those impact on module. So I use the cloning in every controller. For reference. 
https://github.com/LiskHQ/lisk/blob/1.0.0/api/controllers/accounts.js#L40